README.md

```diff
@@ -0,0 +1,193 @@
+> *This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app) +
+  [backpack-react-scripts](https://github.com/Skyscanner/backpack-react-scripts/tree/master/packages/react-scripts).*
+  It is very similar to one that you would create using [Create React App](https://github.com/facebookincubator/create-react-app)
+  without [backpack-react-scripts](https://github.com/Skyscanner/backpack-react-scripts/tree/master/packages/react-scripts),
+  so please refer to [it's documentation](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md)
+  for more usage information.
+
+# [_Put your project name here_]
```

Comment:
Can you update the name and description in here? Also - can you put something in for 'npm install' as I had to run that before npm start would work.

Reply:
Updated readme